_extras/design.md

```diff
@@ -116,7 +116,117 @@ How do I...
 
 ## Stage 2 - Learning Plan
 
-*To Do: point-form outline of the lesson episodes with time estimates for teaching and exercises.*
+### Introductions and Learning Objectives
+
+*   Teaching: 15 min
+*   Exercises: 0 min (accounted for in teaching time - no separate exercise)
+    *   Ensure setup issues resolved
+    *   Assess how much `dplyr` review will be necessary
+
+### Data Management in R
+
+*   Teaching: 10 min
+*   Exercises: 5 min
+    *   Import Gapminder data (`read.csv()`)
+    *   Gapminder data structure (`str()`, `head()`)
+
+### Data Structures
+
+*   Teaching: 5 min
+*   Exercises: 5 min
+    *   Variables and categories
+    *   Values and replicates
+    *   Absolute values vs. relative values
```

Comment:
The 'practical to do' will be identifying these features in the Gapminder data that was just imported. I thought it would be important to introduce the ideas before continually referring back to them throughout the rest of the lesson. For each figure encountered students will walk through the 'Choosing a Good Chart' key decisions, which includes determining the appropriate 'variables and/or categories', assessing the availability of 'values and replicates', and choosing 'absolute vs. relative values'.